tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv

```diff
@@ -5,3 +5,21 @@
 "Az.Compute","Microsoft.Azure.Commands.Compute.SetAzureVMSecurityProfile","Set-AzVMSecurityProfile","0","2020","The cmdlet 'Set-AzVMSecurityProfile' no longer supports the type 'Microsoft.Azure.Management.Compute.Models.SecurityTypes' for parameter 'SecurityType'.","Change the type for parameter 'SecurityType' back to 'Microsoft.Azure.Management.Compute.Models.SecurityTypes'."
 "Az.Compute","Microsoft.Azure.Commands.Compute.Automation.SetAzureRmVmssSecurityProfileCommand","Set-AzVmssSecurityProfile","0","2020","The cmdlet 'Set-AzVmssSecurityProfile' no longer supports the type 'Microsoft.Azure.Management.Compute.Models.SecurityTypes' for parameter 'SecurityType'.","Change the type for parameter 'SecurityType' back to 'Microsoft.Azure.Management.Compute.Models.SecurityTypes'."
 "Az.Compute","Microsoft.Azure.Commands.Compute.Automation.NewAzureCapacityReservation","New-AzCapacityReservation","0","1050","The parameter set '__AllParameterSets' for cmdlet 'New-AzCapacityReservation' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'New-AzCapacityReservation'."
+"Az.Compute","Microsoft.Azure.Commands.Compute.StorageServices.AddAzureVhdCommand","Add-AzVhd","1","8420","Parameter set 'DefaultParameterSet' of cmdlet 'Add-AzVhd' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
```

Comment:
This line should be in SignatureIssues.csv

Reply:
@msJinLei , I updated it.